src/main/java/cs/f10/t1/nursetraverse/logic/commands/ExportCommand.java

```diff
@@ -101,4 +104,42 @@ private boolean indexesAllInBounds(Set<Index> indexes, int bound) {
         }
         return true;
     }
+
+    @Override
+    public boolean equals(Object that) {
+        if (this == that) {
+            return true;
+        } else if (that instanceof ExportCommand) {
+            return this.exportFileName.equals(((ExportCommand) that).exportFileName)
+                    && compareOptionalIndexSet(this.targetIndexes, ((ExportCommand) that).targetIndexes);
+        } else {
+            return false;
+        }
```

Comment:
I think the usual convention for writing equals() is to follow this structure:
```
public boolean equals(Object other) {
        if (other == this) {
            return true;
        }
        if (!(other instanceof ExportCommand)) {
            return false;
        }
        ExportCommand otherCommand = (ExportCommand) other;
        return this.exportFileName.equals(other.exportFileName)
                    && compareOptionalIndexSet(this.targetIndexes, other.targetIndexes);
}
```
i.e. return as soon as possible to avoid `else`, and create a new variable to cast to make it more readable.